using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -204,173 +281,331 @@ ACPTarget.getTntId({tntId in
+var targetParameters = new ACPTargetParameters(null, null, product, order);
+
+ACPTarget.locationClickedWithName("cartLocation", targetParameters);
 ```
+
 {% endtab %}
+
 {% endtabs %}
 
-## Set preview restart deeplink
+## locationsDisplayed
 
-Use this API to set the Target preview URL to be displayed when the preview mode is enabled and preview selections are confirmed.
+Use this API to send a location \(mbox\) display notification to configured Target server. This API should be invoked for a prefetched mbox, after the mbox content is retrieved using [retrieveLocationContent API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#retrieve-location-content-requests). If no previous prefetch request is made and the mbox content is retrieved using `retrieveLocationContent` API, then calling this API does not trigger a notification request to the Target server.
+
+{% hint style="warning" %}
+Do not use this API with the deprecated [loadRequests](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference-deprecated#load-target-requests). For a prefetched mbox, on calling the deprecated `loadRequests` API, mbox display notification is internally sent to the Target server by the SDK.
+{% endhint %}
 
 {% tabs %}
 {% tab title="Android" %}
-### setPreviewRestartDeeplink
 
 #### Syntax
 
-```text
-public static void setPreviewRestartDeepLink(final Uri deepLink)
+```java
+public static void locationsDisplayed(final List<String> mboxNames, final TargetParameters targetParameters)
 ```
 
+* _mboxNames_ is a list of mbox locations for which display notification is to be sent to Target.
+* _targetParameters_ is the configured `TargetParameters` for the request.
+
 #### Example
 
 ```java
-Target.setPreviewRestartDeepLink("myapp://HomePage");
+List<String> purchasedProductIds = new ArrayList<String>();
+purchasedProductIds.add("34");
+purchasedProductIds.add("125"); 
+TargetOrder targetOrder = new TargetOrder("123", 567.89, purchasedProductIds);
+
+TargetProduct targetProduct = new TargetProduct("123", "Books");
+
+TargetParameters targetParameters = new TargetParameters.Builder()
+.parameters(new HashMap<String, String>())
+.profileParameters(new HashMap<String, String>())
+.product(targetProduct)
+.order(targetOrder)
+.build();
+
+List<String> mboxList = new ArrayList<>();
+mboxList.add("mboxName1");
+
+Target.locationsDisplayed(mboxList, targetParameters);
 ```
 {% endtab %}
 
 {% tab title="iOS" %}
-### setPreviewRestartDeeplink
 
 #### Syntax
 
 ```objectivec
-+ (void) setPreviewRestartDeeplink: (nonnull NSURL*) deeplink;
++ (void) locationsDisplayed: (nonnull NSArray<NSString*>*) mboxNames 
+withTargetParameters: (nullable ACPTargetParameters*) targetParameters;
 ```
 
+* _mboxNames_ is a NSArray of mbox locations for which display notification is to be sent to Target.
+* _targetParameters_ is the configured `ACPTargetParameters` for the request.
+
```

Comment:
Please reword the bullet points as follows:
"* *mboxNames* is a NSArray of mbox locations for which the display notification will be sent to Target.
* *targetParameters* is the configured `ACPTargetParameters` for the request.